spec/models/invoice_spec.rb

```diff
@@ -187,4 +187,32 @@
       expect(invoice.subscription_fees(subscription.id)).to eq([fee])
     end
   end
+
+  describe '#recurring_fees' do
+    let(:invoice_subscription) { create(:invoice_subscription) }
+
+    it 'returns the fees of the corresponding invoice_subscription' do
+      invoice = invoice_subscription.invoice
+      subscription = invoice_subscription.subscription
+      billable_metric = create(:recurring_billable_metric, organization: subscription.organization)
+      charge = create(:standard_charge, plan: subscription.plan, billable_metric: billable_metric)
+      fee = create(:charge_fee, subscription: subscription, invoice: invoice, charge: charge)
+
+      expect(invoice.recurring_fees(subscription.id)).to eq([fee])
+    end
+  end
+
+  describe '#recurring_breakdown' do
+    let(:invoice_subscription) { create(:invoice_subscription) }
+
+    it 'returns the fees of the corresponding invoice_subscription' do
+      invoice = invoice_subscription.invoice
+      subscription = invoice_subscription.subscription
+      billable_metric = create(:recurring_billable_metric, organization: subscription.organization)
+      charge = create(:standard_charge, plan: subscription.plan, billable_metric: billable_metric)
```

Comment:
Just a style comment :D Not sure why in this file in general in several more places we don't create variables with `let` but instead we are using local variables?

Reply:
Well, not sure exactly :D
I guess it's because it used only in one single test. But that's true that we  could refact it a bit :+1: